src/styles/icon_close.js

```diff
@@ -0,0 +1,3 @@
+import { css } from 'lit-element';
+
+export default css`data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgNDAgNDAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgYXJpYS1sYWJlbGxlZGJ5PSJjbG9zZVRpdGxlSUQgY2xvc2VEZXNjSUQiPjxwYXRoIGQ9Ik0yOS4xOTIgMTAuODA4YTEuNSAxLjUgMCAwMTAgMi4xMkwyMi4xMjIgMjBsNy4wNyA3LjA3MmExLjUgMS41IDAgMDEtMi4xMiAyLjEyMWwtNy4wNzMtNy4wNy03LjA3IDcuMDdhMS41IDEuNSAwIDAxLTIuMTIxLTIuMTJsNy4wNy03LjA3My03LjA3LTcuMDdhMS41IDEuNSAwIDAxMi4xMi0yLjEyMUwyMCAxNy44NzhsNy4wNzItNy4wN2ExLjUgMS41IDAgMDEyLjEyMSAweiIgY2xhc3M9ImZpbGwtY29sb3IiIGZpbGwtcnVsZT0iZXZlbm9kZCIvPjwvc3ZnPg==`;
```

Comment:
Interesting so this works? I was trying to do something like this for the donation form, except I used `background: url()`, not `mask`. I ended up having some browser-specific problems.. like I think it worked in Safari, but not in Chrome or something like that so I ended up just using inline images.

Reply:
Oh I see you used `mask` in one spot and `background` in another.. they're both working fine?

Reply:
Yeah, they work fine. On the 'X' close icon, I used it as a mask so I could set its color. The close menu icon pictured here uses SVG as a background image, since it's got color data embedded in it.

<img width="640" alt="Screen Shot 2020-08-20 at 2 27 06 PM" src="https://user-images.githubusercontent.com/39553/90810713-694ef780-e2f1-11ea-9b09-9cea0a84a67a.png">